app/styles/join.scss

```diff
@@ -11,8 +11,9 @@
 	}
 
 	.card-img-top {
-		max-height: 228px;
+		height: 228px;
```

Comment:
does this work on mobile devices? For safety i would do both height and max-height then

Reply:
(without max-height your overflow: hidden does nothing)

Reply:
It does work, but took the suggestion of adding max-height and implemented that as well